### ISSUE TITLE
Title and Immediately Following - Web Version

### ISSUE THREAD

**Issue by ERyan71258** [SMPTE/st429-4#2]
Strictly cosmetic: Using a soft return, I would move "MXF" to the next line.

How is the alignment of the SMPTE log determined? It is usually horizontally center-aligned with the title. This was accomplished using a table.

Is 2023-05-04 approval date correct or is it a placeholder?

There is no SMPTE contact information such as address or phone number. Should also include a link to the SMPTE website.

**Comment by palemieux** [comment]
> Is 2023-05-04 approval date correct or is it a placeholder?

Yes, this is the correct since the document was last approved by the TC on that date.
